examples/advanced_operations/add_responsive_search_ad_full.py

```diff
@@ -315,9 +433,7 @@ def add_keywords(client, customer_id, ad_group_resource_name):
     ad_group_criterion.ad_group = ad_group_resource_name
     ad_group_criterion.status = client.enums.AdGroupCriterionStatusEnum.ENABLED
     ad_group_criterion.keyword.text = _KEYWORD_TEXT_EXACT_1
```

Comment:
Not completely necessary but I'm going to create a map of keyword match type to keyword text and use a loop to make this section more DRY